crates/rpc/api/src/lib.rs

```diff
@@ -109,6 +109,10 @@ pub trait StrataApi {
     #[method(name = "getCheckpointInfo")]
     async fn get_checkpoint_info(&self, idx: u64) -> RpcResult<Option<RpcCheckpointInfo>>;
 
+    /// Get the checkpoint status if checkpoint exists
+    #[method(name = "getCheckpointStatus")]
+    async fn get_checkpoint_status(&self, idx: u64) -> RpcResult<Option<RpcCheckpointConfStatus>>;
```

Comment:
I don't get it. Why the type is called `RpcCheckpointConfStatus` and the method/fun is `getCheckpointStatus`/`get_checkpoint_status`?

What am I missing?

Either should be:

- `getCheckpointConfStatus`/`get_checkpoint_conf_status`
- `RpcCheckpointStatus`

Reply:
Also what is `Conf`? `Confirmation`, `Config`?

Reply:
I'm imagining it's "confirmation" but I agree, should be consistent here.